src/Graphics/Texture.cs

```diff
@@ -67,6 +67,42 @@ internal protected override void GraphicsDeviceResetting()
 
 		#region Static SurfaceFormat Size Methods
 
+		private static int GetBlockSizeSquared(SurfaceFormat format)
```

Comment:
Argh, of course I misread the diff, this one's my fault:

```suggestion
		protected static int GetBlockSizeSquared(SurfaceFormat format)
```